lib/Server.js

```
@@ -3412,39 +3412,109 @@ class Server {
+      const prefix =
+        magicIndex === -1 ? pattern : pattern.slice(0, magicIndex);
+      const base = prefix.endsWith("/")
+        ? prefix.slice(0, -1)
+        : path.posix.dirname(prefix);
 
-    const expand = (/** @type {string} */ item) => {
-      const posix = toPosix(item);
-      return isDynamicPattern(posix)
-        ? globSync(posix, { cwd, absolute: true }).map(toNative)
-        : item;
+      return path.resolve(cwd, (base || ".").replace(/\\(.)/g, "$1"));
```

ESLint

[error] Prefer `String#replaceAll()` over `String#replace()`.

(unicorn/prefer-string-replace-all)